samples/suit/smp_transfer/sysbuild/hci_ipc.overlay

```diff
@@ -7,3 +7,23 @@
 &uart135 {
 	status = "disabled";
 };
+
+
+&mram1x {
+	cpurad_rx_partitions: cpurad-rx-partitions {
+		compatible = "nordic,owned-partitions", "fixed-partitions";
+		status = "okay";
+		perm-read;
+		perm-execute;
+		perm-secure;
+		#address-cells = < 0x1 >;
+		#size-cells = < 0x1 >;
+		cpurad_slot0_partition: partition@66000 {
+			reg = < 0x66000 DT_SIZE_K(256) >;
+		};
+
+		recovery_partition: partition@a6000 {
+			reg = < 0xa6000 DT_SIZE_K(100) >;
```

Comment:
The recovery partition size does not match?

Reply:
Discussed offline: this is the `cpurad` recovery partition and the new name makes it obvious.